gamestonk_terminal/stocks/due_diligence/business_insider_view.py

```diff
@@ -116,7 +83,7 @@ def price_target_from_analysts(
         plt.title(f"{ticker} (Time Series) and Price Target")
         plt.xlim(stock.index[0], stock.index[-1])
         plt.xlabel("Time")
-        plt.ylabel("Share Price ($)")
+        plt.ylabel("Share Price")
         plt.grid(b=True, which="major", color="#666666", linestyle="-")
         plt.minorticks_on()
```

Comment:
can we lose the minorticks?  I have been dropping them as my personal preference is they make the plots less nice

Reply:
I thought it looked nice :( will get rid of it